packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -6243,24 +6244,19 @@ describe('plugin-meetings', () => {
           );
 
           // beacuse we are calling callback so we need to wait
+          assert.calledWith(meeting.unpublishStreams,[
```

Comment:
just to double check - are these streams inside mediaProperties actually set to something other than undefined or null in this test?

Reply:
Oh. Looks like they were null. I have pushed a new commit to have dummy strings for them on `beforeEach`. Kindly check.